ui/src/util/valid.tsx

```diff
@@ -0,0 +1,3 @@
+export function isValid<Type>(arg: Type) {
+  return arg !== null && typeof arg !== "undefined";
```

Comment:
I'm curious about the `typeof arg !== "undefined"` check. What cases does this cover beyond the usual `arg !== undefined` ?